test/Core.Test/AdminConsole/OrganizationFeatures/Policies/PolicyRequirements/SendPolicyRequirementTests.cs

```diff
@@ -0,0 +1,126 @@
+﻿using AutoFixture.Xunit2;
+using Bit.Core.AdminConsole.Enums;
+using Bit.Core.AdminConsole.Models.Data.Organizations.Policies;
+using Bit.Core.AdminConsole.OrganizationFeatures.Policies.PolicyRequirements;
+using Bit.Core.Enums;
+using Bit.Core.Test.AdminConsole.AutoFixture;
+using Xunit;
+
+namespace Bit.Core.Test.AdminConsole.OrganizationFeatures.Policies.PolicyRequirements;
+
+public class SendPolicyRequirementTests
+{
+    // DisableSend
```

Comment:
🎨 Consider using a `#region` here instead of a comment. (This applies to other lines in this file, too.)

Reply:
You're right to call me out on this, but for a slightly different reason. Our code style is [not to use region comments](https://contributing.bitwarden.com/contributing/code-style/#code-regions), so here I am... not using a region comment 😁 

I think these comments are probably not very useful anyway, so I've removed them.